install.sh

```diff
@@ -1,25 +1,51 @@
+#!/bin/bash
+
+###############################################################################
+# Instalation Script for Gitshots
+# Repository: https://github.com/ranman/gitshots-client
+# gitshots.com - Take a picture and collect some stats every time you commit!
+###############################################################################
+
+USERHOME=$(dscl . -read /Users/`whoami` NFSHomeDirectory | awk -F": " '{print $2}')
+POST_COMMIT_FILE=${USERHOME}/bin/post-commit.py
+GIT_TEMPLATE_DIR=${USERHOME}/.git_template
+GIT_TEMPLATE_HOOKS_DIR=${GIT_TEMPLATE_DIR}/hooks
+
+printf "\nPutting post-commit.py in bin.\n"
+if [[ -f $POST_COMMIT_FILE ]]; then
+   printf "\nRemoving old file: $POST_COMMIT_FILE\n"
+   rm $POST_COMMIT_FILE
+else
+	mkdir -p ~/bin
+fi
+
+if [[ $1 && -f $1 ]]; then
+   	printf "\nSymbolic link from: $1 to $POST_COMMIT_FILE\n"
+	ln -s $1 $POST_COMMIT_FILE
+else
+	printf "\nDownload from github master branch: $POST_COMMIT_FILE\n"
+	curl https://raw.githubusercontent.com/ranman/gitshots-client/master/post-commit.py > $POST_COMMIT_FILE
+fi
+
+printf "\nCreating git_template dir.\n"
+mkdir -p $GIT_TEMPLATE_HOOKS_DIR
+git config --global init.templatedir $GIT_TEMPLATE_DIR
+
+printf "\nCreating post-commit hook.\n"
+cat << EOF > $GIT_TEMPLATE_HOOKS_DIR/post-commit
 #!/bin/sh
-bin=~/bin
-hooks=~/.git_template/hooks
-
-echo "Creating git_template dir"
-mkdir -p ~/.git_template/hooks
-git config --global init.templatedir '~/.git_template/'
-echo "Putting post-commit.py in bin"
-mkdir -p ~/bin
-curl https://raw.githubusercontent.com/ranman/gitshots-client/master/post-commit.py > ~/bin/post-commit.py
-echo "Creating post-commit hook"
-cat << EOF > ~/.git_template/hooks/post-commit
-#!/bin/sh
-/usr/bin/python2.7 ~/bin/post-commit.py
+/usr/bin/python2.7 $POST_COMMIT_FILE
 EOF
-chmod +x ~/.git_template/hooks/post-commit
+chmod +x $GIT_TEMPLATE_HOOKS_DIR/post-commit
 
-echo "Installing CoreLocationCLI dependency"
-brew install corelocationcli
-echo "Installing imagesnap dependency"
-brew install imagesnap
-echo "Installing requests dependency, if this fails fix your python or use sudo"
-pip install requests
+if [[ !($2 && $2 == "--no-dependencies") ]]; then
```

Comment:
I will take a look in the documentation, it's working like this for both type of installation (script only and homebrew)